app/views/dvr/checkCasesDetails.scala.html

```diff
@@ -54,24 +54,38 @@
     <p>@Html(messages("challengeRules.rules.paper", config.businessRatesValuationFrontendUrl(s"property-link/valuations/startChallenge?backLinkUrl=$returnUrl")))</p>
 }
 
-@row(submittedDate: LocalDateTime, status: String, checkReference: String, closedDate: Option[LocalDate], clientOrAgent: String, submittedBy: String) = @{
+    @tagColour(status: String) = {
+      @status match {
+        case "DECISION SENT" | "CLOSED" | "CANCELLED" => { govuk-tag--grey }
+        case _ => {}
+      }
+    }
+
+@statusTagWithHelpLink(status: String) = {
+  <strong class="govuk-tag @tagColour(status)">@status</strong>
```

Comment:
might be best to extract this svg into another file as it will be used by the challenge tab as well and possibly other tables.

Reply:
Yep. `svg` could be a file, or could be defined in css. Even it isn't reused, it's nicer to extract somewhere.

Reply:
just want to point out that I copy-pasted the help icon from valuation frontend https://github.com/hmrc/business-rates-valuation-frontend/blob/main/app/uk/gov/voa/businessrates/valuation/assessment/views/pages/detailedvaluation/partials/messageWithHelpIcon.scala.html#L13

Reply:
I will extract this to a separate twirl template, rather than CSS or svg file